x-pack/legacy/plugins/apm/cypress/cypress/support/index.d.ts

```diff
@@ -1,9 +1,18 @@
+/// <reference types="Cypress" />
+
 /*
  * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
 
+// typically custom commands are added in this support folder
+// so it makes sense to put their TypeScript definitions here
+// from the JavaScript specs loads this file using
+// the triple slash "reference" comment like this:
+//
+// /// <reference path="../support/index.d.ts" />
+
 declare namespace Cypress {
   interface Chainable<Subject> {
     snapshot: () => {};
```

Comment:
This is the function that is nor defined in the tests 🤷‍♀ How could we include in the tests?

Reply:
this is a type def, it's not the actual code. likely we need to include `cypress/snapshot` somewhere.